src/Pharmacy/Services/Import.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+namespace OpenEMR\Pharmacy\Service;
+
+use Address;
+use OpenEMR\Common\Http\oeHttp;
+use Pharmacy;
+
+/**
+ * Class Import
+ * @package Import
+ * This class extends the Pharmacy class to import pharmacies listed with CMS.
+ * It can be adapted to work in other countries if a similar API is available.
+ * There is a duplication check using the NPI number. If the NPI number exist in the table. The entry is skipped.
+ * However, if the pharmacy gets a new NPI number, there could be two entries with the same address and different
+ * NPI numbers. I have discovered that some times erroneous entries can be retrieved.
+ */
+class Import extends Pharmacy
```

Comment:
Why does this class need to extend the Pharmacy class?

Reply:
In my limited understanding of classes and their relationship. I had a different approach in mind when I started this project. In my understanding, I am extending the functional of the Pharmacy class since it is the class that saves/persist the data to the database.  I had been doing some reading on class extensions. But later as I was building I found that I had access to the pharmacy class by USE Pharmacy. I did not remove the extends. I was not sure which was providing me access to the class methods so I left it there. 
I used import because Robert suggested it.